### DISCUSSION TITLE
Failing on taiko being 'mid-game' instead of at the maps end

### DISCUSSION THREAD

**Discussion by Tenexxt** [ppy/osu#29526] [Ideas]
As a person who tries to play all modes, i have a problem with playing taiko and mania because skill issue not going to lie.
I have problem playing taiko for these reasons:

- Skill issue
- Dying being postponed

while its obviously my fault for dying in the first place, there is no indicator that i died in the first place. The rating icon still shows the rating as normal, and there isn't a any sort of indicatior that suggests you basically failed.
Some would probably would like the failing system to still work like this, but IMO it doesn't make sense for your fail to be shown at the end of the map instead of you being told you died already. I have a big skill issue in taiko, and i'd personally prefer being told i failed already instead of having to hope i passed after i had a streak of misses and later having to be disappointed i died at that or some other part and i continued playing for nothing

What im asking isn't neceseraily making failing work like in other modes, but atleast for there to be a indicator displayed like when you fail in a multiplayer lobby so you know you failed but can play anyway.

**Comment by ChenMerami** [comment]
please no

**Comment by ElektroJake** [comment]
It would be good to at least have an indicator for what the win condition is. You will absolutely not know ***why*** you failed as a new player in taiko. This is especially true when you come from osu!standard.
Is this handled differently in osu stable?

**Reply by Tenexxt** [reply]
Not sure since I play on Lazer only 
If I could I would just delete this discussion since I think making this change is a bad idea now (I'm a actual Taiko player now), at most a mod with this could be made

**Reply by ElektroJake** [reply]
Oh, when I think about it... What you previously meant could actually be a mod similar to what Accuracy Challenge does or maybe even an extension to it. It's more about "health" though. Sudden Death wouldn't really fit too because it's more like a Quicker Death? ;p

**Reply by Tenexxt** [reply]
a new `Health Alteration` mod could be made so you can change taiko to other ruleset HP system and for others into the taiko system, and also allowing to make adjustements to the taiko one (so the requirement could be made 60% instead of 50% for example)

**Reply by huuishuu** [reply]
I've had this same idea that it could be made into an optional mod. If I recall correctly, either in early versions of Lazer or in early iterations of ScoreV2 on Stable - this was made into default before people complained and it was reverted. I'd love to see it as an alternative mod for Taiko, sounds interesting if anything.
